lib/modules/dosomething/dosomething_reportback/includes/ReportbackItemTransformer.php

```diff
@@ -18,6 +18,7 @@ public function index($parameters) {
     }
     catch (Exception $error) {
       // @TODO: Potentially log error to watchdog.
+      http_response_code('404');
```

Comment:
@chloealee Since it's an index endpoint, it's still technically "successful" even if it doesn't return anything. Whereas asking for a specific thing that doesn't exist is probably an error... i.e. the difference between going to your email inbox and seeing zero messages vs. going to a specific message and it not existing.